### ISSUE TITLE
Playing sounds problem

### ISSUE THREAD

**Issue by danpal96** [pvcraven/arcade#270]
## Bug Report

System:
Ubuntu 18.04
Python 3.6.5 |Anaconda custom (64-bit)| (default, Apr 29 2018, 16:14:56) [GCC 7.2.0] on linux
arcade 1.3.5 (pip)

I was trying to play sounds while doing the chapter 1.14.2 of the book, but I couldn't.
I tried with the laser.ogg file but the next error occurred:

```
  File "/home/danpal/anaconda3/envs/arcade/lib/python3.6/site-packages/arcade/sound.py", line 172, in _playsound_unix
    if isinstance(sound, pyglet.media.sources.riff.WaveSource):
AttributeError: module 'pyglet.media.sources' has no attribute 'riff'
```

Then I changed this in sound.py:

```
from pyglet.media.sources.riff import WaveSource
def _playsound_unix(sound):
    if` isinstance(sound, WaveSource):
```
But the next error ocurred:
```
but the Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'gi'
```
Then installed pygobject with conda, but the next error ocurred:
```
  File "/home/danpal/anaconda3/envs/arcade/lib/python3.6/site-packages/arcade/sound.py", line 181, in _playsound_unix
    gi.require_version('Gst', '1.0')
  File "/home/danpal/anaconda3/envs/arcade/lib/python3.6/site-packages/gi/__init__.py", line 130, in require_version
    raise ValueError('Namespace %s not available' % namespace)
ValueError: Namespace Gst not available
```
I didn't know what to do here, so i tried converting laser.ogg to wav, but if I play the sound more than one time,  like in the next example, then an error ocurred.

```
import arcade

def run():
    arcade.open_window(300, 300, "Demo Sound")
    laser = arcade.load_sound('/home/danpal/laser.wav')
    arcade.play_sound(laser)
    arcade.play_sound(laser)
    arcade.run()
    sound.play()
run()
```
```
  File "/home/danpal/anaconda3/envs/arcade/lib/python3.6/site-packages/arcade/sound.py", line 174, in _playsound_unix
  File "/home/danpal/anaconda3/envs/arcade/lib/python3.6/site-packages/pyglet/media/sources/base.py", line 269, in play
    player.queue(self)
  File "/home/danpal/anaconda3/envs/arcade/lib/python3.6/site-packages/pyglet/media/player.py", line 100, in queue
    group.queue(source)
  File "/home/danpal/anaconda3/envs/arcade/lib/python3.6/site-packages/pyglet/media/sources/base.py", line 499, in queue
    source = source._get_queue_source()
  File "/home/danpal/anaconda3/envs/arcade/lib/python3.6/site-packages/pyglet/media/sources/base.py", line 377, in _get_queue_source
    raise MediaException('This source is already queued on a player.')
pyglet.media.exceptions.MediaException: This source is already queued on a player.
```
finally, following the recommendation in the next link:
[https://groups.google.com/forum/#!topic/pyglet-users/S8DD16bYzAA](url)
I added streaming=False as an argument to pyglet.media.load in sounds.py
```
def _load_sound_unix(filename: str) -> typing.Any:
    if filename.endswith(".wav"):
	# my_sound = pyglet.media.load(filename)
        my_sound = pyglet.media.load(filename, streaming=False)
        return my_sound
    else:
        return filename
```
and had to change the if statement in _playsound_unix to always evaluate to True
```
def _playsound_unix(sound):
    # pathname2url escapes non-URL-safe characters
    # if isinstance(sound, WaveSource):
    if True:
        sound.play()
        return

```
And now i can play sound but only in wav format.

**Comment by pvcraven** [comment]
gi is a package that is often confused with another package called gi that only installs the needed libraries into the system Python. Not Anaconda or any other downloaded python.

I'm hoping the recent work with ffmpeg for version 2.0 of Arcade will help. But I'm waiting on Pyglet to implement their part. 

So no solution yet, but maybe in a few months.

**Comment by pvcraven** [comment]
Should work better with ffmpeg in next release.

**Comment by pvcraven** [comment]
Closed with 2.0 release.
